backend/src/main/java/com/chinguv45tier3team53/tripplannerrest/entity/User.java

```diff
@@ -0,0 +1,73 @@
+package com.chinguv45tier3team53.tripplannerrest.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "user")
```

Comment:
user might be a reserved word in mysql

Reply:
so when I ran the program, it didn't face any problems, so it seems to be okay
but I'll change the table name to "_user" just to be safe